### PR TITLE
Add ability to self assess satisfaction

### PR DESCRIPTION
**Pull request by kdav108**
## Description
Follows the same structure as #1 and allows developers to also rate their satisfaction for a given card from the Issue panel.

## Screenshots
Screen Shot 2022-09-18 at 7 05 23 pm

Screen Shot 2022-09-18 at 7 05 35 pm

### CODE REVIEW

**Comment by kdav108** [comment]
I'll rebase this PR once #1 is merged in.

File: src/satisfaction.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+};
+
+const MySatisfactionEdit = () => {
+  const onSubmit = (formValue) => {
+      return formValue.myPerformanceRating;
+  }
+
+  return (
+      <CustomFieldEdit onSubmit={onSubmit} header="How would you rate your satisfaction of this Issue?" width="medium" >
+          <Select label="Select one of the options below" name="myPerformanceRating">
```
Comment (ssha0054) [inline src/satisfaction.js]: Can change name from `myPerformanceRating` to `mySatisfactionRating`.

**Review by ssha0054 — APPROVED** [review]
Tested on my dev instance and satisfaction ratings work as expected.

**Comment by kdav108** [comment]
Have rebased on top of current master and tested on my local instance. Could you guys test it once from your end as well? @ssha0054 @andrwcao

**Review by ssha0054 — APPROVED** [review]
Works well on my end!

**Pull request merged**